ui/appui-abstract/src/appui-abstract/quantity/BaseQuantityDescription.ts

```diff
@@ -20,10 +20,13 @@ export abstract class BaseQuantityDescription implements PropertyDescription {
   public displayLabel: string;
   public typename: string;
   public editor: PropertyEditorInfo;
+  public kindOfQuantityName?: string;
 
-  constructor(name: string, displayLabel: string, iconSpec?: string) {
+
+  constructor(name: string, displayLabel: string, iconSpec?: string, kindOfQuantityName?: string) {
```

Comment:
Do we intend to deprecate `BaseQuantityDescription`? If so - asking consumers to provide a `kindOfQuantityName` arg will lead to one more change in the near future. I.w. would a helper to create `PropertyDescription` work better?

Reply:
This warrants more discussion later on, I believe it'll impact many of the tools that are using this class. This class feels like the most basic implementation of the `PropertyDescription` interface, I'm not sure a helper for `PropertyDescription` is worth it when it's just an interface. A helper to create an instance of `BaseQuantityDescription` might help? I would leave this to you and the presentation side.